kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/test/RoleAccessTest.java

```diff
@@ -66,11 +66,17 @@ public void setUp() throws IdUsedException, IdInvalidException, InconsistentExce
         session.setUserId("admin");
 
         _ss = (SiteService)getService(SiteService.class.getName());
-        Site newSite = _ss.addSite(SITE_ID, (String) null);
-        Group group = newSite.addGroup();
-        group.setTitle(".group");
-        _groupReference = group.getReference();
-        _ss.save(newSite);
+        Site newSite = null;
```

Comment:
Thanks, I'll take the first option but it's nice to know about the second
one.

2016-06-27 16:27 GMT+02:00 Matthew Buckett <notifications@github.com>:

> In
> kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/test/RoleAccessTest.java
> <https://github.com/sakaiproject/sakai/pull/2925#discussion_r68585545>:
>
> > @@ -66,11 +66,17 @@ public void setUp() throws IdUsedException, IdInvalidException, InconsistentExce
> >          session.setUserId("admin");
> >
> >          _ss = (SiteService)getService(SiteService.class.getName());
> > -        Site newSite = _ss.addSite(SITE_ID, (String) null);
> > -        Group group = newSite.addGroup();
> > -        group.setTitle(".group");
> > -        _groupReference = group.getReference();
> > -        _ss.save(newSite);
> > +        Site newSite = null;
>
> Just add a second remove call to the tests so it becomes
>
> // Mark as deleted.
> siteService.remoteSite(site);
> // Remove the deleted site.
> siteService.remoteSite(site);
>
> or add specific config for the test to disable the site recycle bin
>
> @BeforeClass
> public static void beforeClass() {
>     try {
>         oneTimeSetup("disable-site-recycle");
>
> and then create a sakai.properties file in the disablesite-recycle folder
> that disables the site recycle bin. You can see an example of how this is
> done with the disable_user_cache configuration for tests.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sakaiproject/sakai/pull/2925/files/c348ce39511520ae4ce640595a668577db967d06#r68585545>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AK6IlkSWQ-MH3UG7w39dMWo7dJCag86Rks5qP93EgaJpZM4I9pie>
> .
>